x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlQueryRequest.java

```diff
@@ -43,17 +47,19 @@
      * See {@code SqlTranslateRequest.toXContent}
      */
     private Boolean columnar = Boolean.FALSE;
+    private boolean fieldmultiValueLeniency = Protocol.FIELD_MULTI_VALUE_LENIENCY;
 
     public SqlQueryRequest() {
         super();
     }
 
     public SqlQueryRequest(String query, List<SqlTypedParamValue> params, QueryBuilder filter, ZoneId zoneId,
                            int fetchSize, TimeValue requestTimeout, TimeValue pageTimeout, Boolean columnar,
-                           String cursor, RequestInfo requestInfo) {
+                           String cursor, RequestInfo requestInfo, boolean multiValueFieldLeniency) {
         super(query, params, filter, zoneId, fetchSize, requestTimeout, pageTimeout, requestInfo);
         this.cursor = cursor;
         this.columnar = columnar;
+        this.fieldmultiValueLeniency = multiValueFieldLeniency;
```

Comment:
Why the different naming style between the constructor parameter and the inner variable? `fieldmultiValueLeniency` VS. `multiValueFieldLeniency`. Also, if you want to stick with this difference, it should be `fieldMultiValueLeniency` (capital letter M).

Reply:
Rename leftover.